packages/snap/jest.config.ts

```diff
@@ -37,7 +37,7 @@ export default {
     uint8arrays:
       '<rootDir>/../../node_modules/.pnpm/uint8arrays@4.0.3/node_modules/uint8arrays/src/index.ts',
     '@ipld/dag-cbor':
-      '<rootDir>/../../node_modules/.pnpm/@didtools+cacao@2.1.0/node_modules/@ipld/dag-cbor/dist/index.min.js',
+      '<rootDir>/../../node_modules/.pnpm/@didtools+cacao@2.0.0/node_modules/@ipld/dag-cbor/dist/index.min.js',
```

Comment:
Not sure about this one, I think develop has version 2.1.0. Just mentioning it. You can leave it as is and if the CI fails on develop, we can fix it in a short follow up PR